src/bootstrap/src/core/builder/mod.rs

```diff
@@ -1212,9 +1206,21 @@ impl<'a> Builder<'a> {
                 sysroot
```

Comment:
Remark [COM 2/3]: ... and return the `rustlib` "parent" instead of `rustlib/$target/lib`,